packages/executor/src/executor.ts

```diff
@@ -463,22 +501,22 @@ export class BasicExecutor implements IExecutor {
         let txHash: HexData32
         try {
             txHash = await ep.write.handleOps(
-                [opsToBundle.map((op) => op.userOperation), wallet.address],
+                [opsToBundle.map((op) => (op.userOperation as UserOperation)), wallet.address],
```

Comment:
why do we need to cast this? we shouldn't have to, right?

Reply:
Bad variable naming on my end, op is of type UserOperationWithHash, the field op.userOperation is of the union type MempoolUserOperation. renamed UserOperationWithHash's userOperation field to mempoolUserOperation